All CUDA 13 wheel-testing jobs are failing like this:

```text
=========================== short test summary info ============================
FAILED python/kvikio/tests/test_benchmarks.py::test_single_node_io[cufile] - assert 1 == 0
FAILED python/kvikio/tests/test_benchmarks.py::test_single_node_io[posix] - assert 1 == 0
FAILED python/kvikio/tests/test_benchmarks.py::test_single_node_io[cufile-mfma] - assert 1 == 0
FAILED python/kvikio/tests/test_benchmarks.py::test_single_node_io[cufile-mf] - assert 1 == 0
FAILED python/kvikio/tests/test_benchmarks.py::test_single_node_io[cufile-ma] - assert 1 == 0
============ 5 failed, 1512 passed, 13 skipped in 105.94s (0:01:45) ============
```

([build link](https://github.com/rapidsai/kvikio/actions/runs/26605052458/job/78421861281?pr=970#step:13:2207))

CUDA 12 jobs are passing.

I'll go ask for some help on that.

--- a/python/kvikio/kvikio/benchmarks/utils.py
+++ b/python/kvikio/kvikio/benchmarks/utils.py
@@ -20,7 +20,7 @@ has_cuda_core_system = False
 with contextlib.suppress(ImportError):
     from cuda.core import system
 
-    has_cuda_core_system = system.CUDA_BINDINGS_IS_NVML_COMPATIBLE
+    has_cuda_core_system = system.CUDA_BINDINGS_NVML_IS_COMPATIBLE
 
 
 def drop_vm_cache() -> None:

--- a/python/kvikio/tests/conftest.py
+++ b/python/kvikio/tests/conftest.py
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: Copyright (c) 2022-2025, NVIDIA CORPORATION. All rights reserved.
+# SPDX-FileCopyrightText: Copyright (c) 2022-2026, NVIDIA CORPORATION. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
 import contextlib

--- a/python/kvikio/tests/conftest.py
+++ b/python/kvikio/tests/conftest.py
@@ -21,13 +21,14 @@ def command_server(conn: Connection) -> None:
         cmd, cwd, verbose = conn.recv()
         # Run command
         res: subprocess.CompletedProcess = subprocess.run(
-            cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, cwd=cwd
+            cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            cwd=cwd,
+            text=True,
         )  # type: ignore
-        if verbose:
-            print(f"{cwd}$ " + " ".join(res.args))
-            print(res.stdout.decode(), end="")
-        # Send return code back to client
-        conn.send(res.returncode)
+        # Send process result back to client.
+        conn.send((res.args, cwd, res.returncode, res.stdout, res.stderr))
 
 
 @pytest.fixture(scope="session", autouse=True)
@@ -49,7 +50,18 @@ def run_cmd():
 
     def run_cmd(cmd: Iterable[str], cwd, verbose=True):
         client_conn.send((cmd, cwd, verbose))
-        return client_conn.recv()
+        args, cwd, returncode, stdout, stderr = client_conn.recv()
+
+        if verbose or returncode != 0:
+            print(f"{cwd}$ " + " ".join(args))
+            if stdout:
+                print("--- stdout ---")
+                print(stdout, end="" if stdout.endswith("\n") else "\n")
+            if stderr:
+                print("--- stderr ---")
+                print(stderr, end="" if stderr.endswith("\n") else "\n")
+
+        return returncode
 
     yield run_cmd
 

